pipeline/scripts/ci/reimage-octo-node.sh

```diff
@@ -75,7 +75,7 @@ for node in ${NODES} ; do
     done
 
     echo 'Setting the systems to use shortnames'
-    ssh ${node} 'sudo hostnamectl set-hostname $(hostname -s)'
+    ssh ${node} 'sudo hostnamectl set-hostname $(hostname -s).ceph.redhat.com'
```

Comment:
Do we need reboot after setting hostname?

Reply:
I think reboot is not required.

Reply:
correct.